compiler-rt/test/asan/TestCases/use-after-free-symbolizer-markup.cpp

```diff
@@ -0,0 +1,33 @@
+// COM: End to end test for the sanitizer symbolizer markup. Since it uses debug info
+// COM: to do offline symbolization we only check that the current module is correctly symbolized
+// REQUIRES: linux
+// RUN: %clangxx_asan %s -Wl,--build-id=0x12345678 -o %t.main
+// RUN: mkdir -p %t/.build-id/12
```

Comment:
```suggestion
// RUN: rm -rf %t
// RUN: mkdir -p %t/.build-id/12
// RUN: %clangxx_asan %s -Wl,--build-id=0x12345678 -o %t/main
```
Its normal to remove the old test dir before doing trying to make a new one or add to it. It also makes sense to put any artifacts into the temp dir you're going to remove, so each run is self contained.

You'll need to adjust the other commands to match.

Reply:
Done